src/taipy/core/data/data_node.py

```diff
@@ -221,6 +221,10 @@ def name(self, val):
     def version(self):
         return self._version
 
+    @version.setter
+    def version(self, val):
+        self._version = val
+
```

Comment:
Here you are exposing the possibility for a user to modify manually the version of an entity. Isn't it too risky? Can we keep it hidden or private?

Reply:
Oh, we are exposing it on `dev/3.0` too.

We ofcourse can remove the setter and just use `_version`.

Reply:
Or calling a method named `_set_version` that is hidden or private.

Something like:
```
def _set_version(self, val):
    self._version = val
    DataManagerFactory.build_manager().set(self)
```

The purpose is just to avoid exposing it to the user.

Reply:
Oh, right. 
So the method is even simpler.

Reply:
What I meant was that there is no need for a method. Just `entity._version`.
Is there any other constrain?

Reply:
Absolutely. You are right!

Reply:
We will need to keep this in mind to remove the version setter from dev/3.0. I will take note and do this latter.